website/package.json

```diff
@@ -42,16 +43,16 @@
   },
   "devDependencies": {
     "@motiz88/react-native-docgen": "0.0.3",
-    "front-matter": "^2.3.0",
-    "fs-extra": "^5.0.0",
-    "glob": "^7.1.2",
-    "glob-promise": "^3.3.0",
-    "he": "^1.2.0",
```

Comment:
@Simek, Can you please verify why `he` is removed. This was being used here https://github.com/MLH-Fellowship/react-native-website/blob/6e41a04f987b8161baaa578bb3334857e4896348/website/scripts/sync-api-docs/propFormatter.js#L38